src/layout/Table/TableComponent.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+
+import { Delete as DeleteIcon } from '@navikt/ds-icons';
+
+import { AppTable } from 'src/app-components/table/Table';
+import { Caption } from 'src/components/form/caption/Caption';
+import { FD } from 'src/features/formData/FormDataWrite';
+import { useDataModelBindings } from 'src/features/formData/useDataModelBindings';
+import { Lang } from 'src/features/language/Lang';
+import { useLanguage } from 'src/features/language/useLanguage';
+import { useIsMobile } from 'src/hooks/useDeviceWidths';
+import { useNodeItem } from 'src/utils/layout/useNodeItem';
+import type { TableActionButton } from 'src/app-components/table/Table';
+import type { PropsFromGenericComponent } from 'src/layout';
+import type { IDataModelReference } from 'src/layout/common.generated';
+import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+
+type TableComponentProps = PropsFromGenericComponent<'Table'>;
+
+type TableSummaryProps = {
+  componentNode: LayoutNode<'Table'>;
+};
+
+export function TableSummary({ componentNode }: TableSummaryProps) {
+  const item = useNodeItem(componentNode);
```

Comment:
You're only using `item.dataModelBindings`, `item.textResourceBindings` and `item.columnConfig` here, so it might be more performant to just select those individually.